src/rule/index.ts

```diff
@@ -59,15 +61,60 @@ const incrementHit = (node: { x_x_x_x_results: a.ReportResult }) => {
   node.x_x_x_x_results.hits++;
 };
 
-const removeServer = (api: a.Root, path: string) => {
+// Named sloppy since there are many cases this function does not catch
+const isAbsoluteUrlSloppy = (urlString: string) => {
+  return (
+    urlString.indexOf("http://") === 0 || urlString.indexOf("https://") === 0
+  );
+};
+
+export const removeServer = (
```

Comment:
This function is now two different non-overlapping execution paths depending on whether the server url is absolute or not. I think it would be better if the big if statement was made into a separate function.